src/json/open_terminal_cmd_opt_t.json.js

```diff
@@ -0,0 +1,50 @@
+// JavaScript should be written in ECMAScript 5.1.
+
+function main() {
+
+    const rules = [
+        {
+            description: "⌘ + ⌥ + T opens Terminal",
+            manipulators: [
+                {
+                    type: "basic",
+                    from: {
+                        key_code: "t",
+                        modifiers: {
+                            mandatory: [
+                                "left_command",
+                                "left_option"
+                            ]
+                        }
+                    },
+                    to: [
+                        {
+                            shell_command: "osascript -e 'tell application \"Terminal\" to do script \"\"'"
```

Comment:
[Tips] I think using open_application feels more natural.
https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/software_function/open_application/
```suggestion
                            "software_function": {
                                "open_application": {
                                    "bundle_identifier": "com.apple.Terminal"
                                }
                            }
```